pymc/sampling_jax.py

```diff
@@ -264,5 +264,6 @@ def sample_numpyro_nuts(
         dims=dims,
         **idata_kwargs,
     )
+    az_trace.sample_stats.attrs["sampling_time"] = (tic3 - tic2).total_seconds()
```

Comment:
Do we want to call it "sampling_time_posterior" so that we know which sampling it is referring to? I suppose we don't care about the sampling time of prior/posterior checks, so probably fine like this.

Reply:
I chose this because it is the same as the default PyMC samplers.